<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheatsView</class>
 <widget class="QWidget" name="CheatsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cheats</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1" colspan="2">
    <widget class="QPushButton" name="remove">
     <property name="text">
      <string>Remove</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QPushButton" name="addSet">
     <property name="text">
      <string>Add New Set</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2">
    <widget class="QPushButton" name="addGSA">
     <property name="text">
      <string>Add GameShark</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QPushButton" name="addPAR">
     <property name="text">
      <string>Add Pro Action Replay</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QPushButton" name="addCB">
     <property name="text">
      <string>Add CodeBreaker</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="load">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QPushButton" name="add">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="save">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="10">
    <widget class="QTreeView" name="cheatList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::MoveAction</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1" rowspan="2" colspan="2">
    <widget class="QPlainTextEdit" name="codeEntry">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cheatList</tabstop>
  <tabstop>addSet</tabstop>
  <tabstop>load</tabstop>
  <tabstop>save</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>codeEntry</tabstop>
  <tabstop>add</tabstop>
  <tabstop>addGSA</tabstop>
  <tabstop>addPAR</tabstop>
  <tabstop>addCB</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
